db/migrate/20170816130953_create_hours.rb

```diff
@@ -2,8 +2,8 @@ class CreateHours < ActiveRecord::Migration[5.1]
   def change
     create_table :hours do |t|
       t.date :meeting_date
-      t.integer :hours
-      t.integer :minutes
+      t.integer :hours, default: 0
+      t.integer :minutes, default: 0
```

Comment:
You should use a new migration to change the default value, since the table already existed for some time.